README.md

```diff
@@ -1 +1,9 @@
-# hello-world
\ No newline at end of file
+# hello-world
+
+Top 5 Rappers
+
+1. JCole
+2. Kendrick Lamar
```

Comment:
Should we replace Kendrick? He hasn't come out with an album in years.

Reply:
Good point. I replaced Kendrick with Travis. Let me know what you think!

Reply:
Good choice!